_posts/en/newsletters/2022-06-15-newsletter.md

```diff
@@ -0,0 +1,286 @@
+---
+title: 'Bitcoin Optech Newsletter #204'
+permalink: /en/newsletters/2022/06/15/
+name: 2022-06-15-newsletter
+slug: 2022-06-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Continued package relay BIP discussion:** a recent draft BIP for
+  [package relay][topic package relay] (see [Newsletter #201][news201
+  relay]) has received additional comments in the past several weeks:
+
+    - *Policy limits:* Anthony Towns [asked][towns relay] if the
+      negotiation between two peers to support package relay should
+      include information about each peer's package maximum size and
+      depth limits, otherwise nodes with non-default settings could
+      receive repeated notifications about packages they were did not
+      want, wasting bandwidth.  BIP author Gloria Zhao [suggests][zhao
+      negotiation] using that the first version of the package relay
+      protocol should imply a maximum package size of 25 transactions
+      and 101,000 vbytes.
+
+    - *Package graph announcement only:* Eric Voskuil
+      [recommends][voskuil graph] that a peer who learns about a
+      high-feerate descendant of a low-feerate ancestor should simply
+      inform its peers of the relationships between those two
+      transactions, called the package graph.  A receiving peer can then
+      request any transactions it doesn't have.  In a separate part of
+      the thread, Towns [notes][towns graph] that a graph can't be
+      validated until all transactions have been received, so care must
+      be taken to ensure a peer can't lie about a graph in order to
+      prevent a transaction from being relayed by other peers.
+
+    - *Using short ids:* several developers suggested using
+      [BIP152][]-style short identifiers (ids).  Zhao [explained][zhao
+      sids] that short ids make sense for block relay where nodes first
+      validate a new block's proof of work (which is expensive to
+      create), so it would be expensive for an attacker to abuse the
+      mechanism to waste node resources.  But, for relay where no
+      significant expense is involved, short ids can be abused over and
```

Comment:
"where no significant expense is involved could perhaps be a bit more specific. How about: 

```suggestion
      mechanism to waste node resources.  But, for relay of data that is not 
      expensive to create, short ids can be abused over and
```